### DISCUSSION TITLE
Install not working with homebrew

### DISCUSSION THREAD

**Discussion by Olivier-Berg** [aedanbricker/quicknav#26] [Q&A] [answered]
When running `brew tap MrDogeBro/quicknav && brew install quicknav` I get an error saying 
```
Error: SHA256 mismatch
Expected: 79be6e58c426cdcad8ee844743f5398c638f40f6504644094f9ee0b72297ce2we
  Actual: 79be6e58c426cdcad8ee844743f5398c638f40f6504644094f9ee0b72297ce2e
    File: /Users/oli4/Library/Caches/Homebrew/downloads/5e292a724dfdf7ad6117963db3cd01795f80a1a482476575f1f56941d48f84e6--quicknav-1.1.1.tar.gz
To retry an incomplete download, remove the file above.
```
So I remove the file and rerun but I get the same error.

For now I installed it using cargo, but you might want to fix it.

**Comment by aedanbricker** [comment, marked as answer]
@IkBenOlie5 thanks for reporting that. It looks like I somehow just added a `w` to the sha256 number. That has now been updated to be the correct sha256 number and should be all working. Thanks!
